ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/efa-pg/provision/cucushift-installer-rehearse-aws-ipi-efa-pg-provision-chain.yaml

```diff
@@ -0,0 +1,24 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-efa-pg-provision
+  steps:
+  - chain: ipi-conf-aws
+  - ref: ipi-conf-aws-custom-region
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  env:
+  - name: CONTROL_PLANE_INSTANCE_TYPE
+    default: "m6i.xlarge"
+    documentation: "Instance type for control plane nodes"
+  - name: COMPUTE_NODE_TYPE
+    default: "m5.xlarge"
+    documentation: "Instance type for compute nodes"
+  - name: ENABLE_EFA_PG_INSTANCE
```

Comment:
It's better to add a description, especially note that `c5n.9xlarge` will be used.

Reply:
Ok, let me add a description.